Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc

```diff
@@ -119,7 +119,7 @@ TH1F* MuonME0DigisHarvestor::ComputeBKG(TH1F* hist1, TH1F* hist2, std::string na
     }
 
     int nEvts = hist2->GetEntries();
-    float scale = 6*2*nEvts*9*25e-9;
+    float scale = 6*2*nEvts*3*25e-9;
```

Comment:
6 layers, 2 endcap regions, we are saving hits only from 3 BXs (-1,0,+1) of 25 ns (before it was 9)